backend_new/prisma/seed-helpers/listing-factory.ts

```diff
@@ -348,7 +492,8 @@ export const listingFactory = (
   ),
 });
 
-const unitFactory = (
+// Temporarily keeping this for reference
+const unitFactory2 = (
```

Comment:
do we still need this since we have a dedicated unit factory?

Reply:
I was keeping this here so merge conflicts were easier every time I pulled in your changes. I can remove it now that it is ready!

Reply:
@YazeedLoonat This has been deleted